client-js/app/elements/labrad-plot.ts

```diff
@@ -1,9 +1,17 @@
 import 'd3';
-
+import * as three from 'three';
 import {viridisData} from '../scripts/colormaps';
 import * as datavault from "../scripts/datavault";
 
+/**
+ * Import the THREE namespace from the module.
+ */
+const THREE = three.default;
```

Comment:
I agree it's not ideal to keep these local changes, but this is the reason why we've got the `.d.ts` definition files checked into the repo, rather than installing them using one of the tools to manage typescript definitions (originally it was `tsd`, then `typings`, now the latest thing is to install types packages from npm like [`@types/three`](https://www.npmjs.com/package/@types/three), who knows what it will be next month...). In practice we don't update these definitions very often, but if we do make an update and forget to change the module declaration, at least we get compiler errors to remind us to fix it :-)